tcf_website/templates/search/search.html

```diff
@@ -72,10 +72,43 @@ <h5><a href="{% url 'department' dept_id=dept_data.dept_id %}">{{ dept }} / {{ d
                               </a>
                             </li>
                         {% endfor %}
-                    </ul>
+                    </ul>-->
+                    <h5><a href="{% url 'department' dept_id=dept_data.dept_id %}">{{ dept }} / {{ dept_data.subdept_name }}</a></h5>
+                      {% for c in dept_data.courses %}
+                        {% if c != dept_data.courses.0 %}
+                        {% endif %}
+                        <div class="list-group-item list-group-item-light text-dark p-2">
+                          <div class="row">
+                            <div class = "col-md-11 align-self-center">
+                                <a href="{% url 'course' mnemonic=c.mnemonic.split.0 course_number=c.number %}">
+                                  <div class = "row g-0 ">
+                                    <div class = "col-md-3">{{c.mnemonic}}</div>
+                                    <div class = "col_md_8">{{c.title}}</div>
+                                  </div>
+                                </a>
+                            </div>
+                            <div class = "col-md-1 align-self-center">
+                              <h5 class="hover-tcf-orange" data-toggle="collapse"
+                                data-target="#description_{{forloop.counter}}"
+                                id="result_{{forloop.counter}}"><i class="fas fa-chevron-down"></i></h5>
+                            </div>
+                          </div>
+                          <div class = "collapse" id="description_{{forloop.counter}}">
```

Comment:
Probably fine here but try to be a bit more specific with your ID names. The given id could probably be used for a lot of different elements on this page so it could create conflict in the future. For this, something like "description_container_{{forloop.counter}}" may be more fitting.